### DISCUSSION TITLE
What is the native alternative for fluent-nhibernate

### DISCUSSION THREAD

**Discussion by danilobreda** [nhibernate/nhibernate-core#2531] [General] [answered]
I believe that many nhibernate developers know about [FluentNhibernate ](https://github.com/FluentNHibernate/fluent-nhibernate)at least in some project.

I use it in several of my projects, but lately, [the project is dying](https://github.com/FluentNHibernate/fluent-nhibernate/issues/459), due to the lack of support from the creators of the project.

We have a lot of problems using the latest versions of nhibernate due to lack of maintenance.
Here are some sample issues and pullrequests.
https://github.com/FluentNHibernate/fluent-nhibernate/issues/430
https://github.com/FluentNHibernate/fluent-nhibernate/issues/429
https://github.com/FluentNHibernate/fluent-nhibernate/issues/456
https://github.com/FluentNHibernate/fluent-nhibernate/pull/432
https://github.com/FluentNHibernate/fluent-nhibernate/pull/453

Could you help us with that? Maybe migrating/fork the repository to the [nhibernate github "group"](https://github.com/nhibernate), or what alternatives do we have?

**Comment by fredericDelaporte** [comment]
Hello,

First, we intend issues on Github to be about bugs and feature requests, and we recommend posting on [groups](https://nhibernate.info/groups.html) for discussions or support.

This said, #2371 happens to tell about mappings API available in NHibernate, so have a look there.

About forking fluent-nhibernate, eventually in the nhibernate repository, this really should be discussed on the groups (likely on the [dev one](https://groups.google.com/g/nhibernate-development)). (If you do so, I think you should clarify in your post on groups if the trouble is a lack of reviewers with merge right and ability to release, or also if contributors are lacking ; how you intend to release if you cannot get authorization on the current NuGet package, ...)

**Comment by danilobreda** [comment]
Thanks for the way so that we community understand what to do.  Posted on the dev one.

**Comment by hazzik** [comment, marked as answer]
Fluent NHibernate has joined NHibernate. Hopefully we'll be able to revive it.
